plugins/modules/azure_rm_postgresqlserver.py

```diff
@@ -72,17 +86,34 @@
     admin_username:
         description:
             - The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation).
+        type: str
     admin_password:
         description:
             - The password of the administrator login.
+        type: str
     create_mode:
         description:
-            - Create mode of SQL Server.
-        default: Default
+            - Create mode of SQL Server. Blank (default), restore from geo redundant (geo_restore), or restore from point in time (point_in_time_restore).
+        type: str
+        default: default
+        choices:
+            - default
+            - geo_restore
+            - point_in_time_restore
+    source_server_id:
+        description:
+            - Id if the source server if I(create_mode=default).
```

Comment:
@karisN There is something wrong with this description statement. Please refer to the relevant documents of this module and describe it again. Thank you very much!